src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -1086,6 +1086,9 @@ namespace WaterToAirHeatPumpSimple {
 								MixHumRat = OutAirFrac * FinalSysSizing( CurSysNum ).PrecoolHumRat + ( 1.0 - OutAirFrac ) * FinalSysSizing( CurSysNum ).RetHumRatAtCoolPeak;
 							}
 						}
+						// supply air condition is capped with that of mixed air to avoid SHR > 1.0
+						SupTemp = min( MixTemp, SupTemp );
+						SupHumRat = min( MixHumRat, SupHumRat );
 						OutTemp = FinalSysSizing( CurSysNum ).OutTempAtCoolPeak;
 						rhoair = PsyRhoAirFnPbTdbW( StdBaroPress, MixTemp, MixHumRat, RoutineName );
 						MixEnth = PsyHFnTdbW( MixTemp, MixHumRat );
```

Comment:
@mjwitte @Myoldmopar Is there an easy way to check space? I have to commit a modified unit test for this issue, so  I can clean it before I re-commit the unit test update. I manually checked for space where I made changes but couldn't fine one. Am I missing some thing here?